### ISSUE TITLE
What is the mAP on COCO17 val ?

### ISSUE THREAD

**Issue by Tyler-D** [yyccR/yolov5_in_tf2_keras#4]
Hi @yyccR, thanks for your repo. I want to know if you can reach the same mAP as in original YOLOV5 (Train on COCO17 train and test on COCO17 val)? And do you have plan to release some pretrained checkpoint ?

**Comment by yyccR** [comment]
That's what I've always wanted to do，but my machine was busy and I didn't have time to train the total COCO2017 dataset, which seemed to take a few days to converge。So reaching the same mAP is my goal, and I will release pre-trained checkpoints once it reaches a better accuracy.
